.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,18 @@
+- id: wazo-copyright-check
```

Comment:
Could you add a comment at the top of the files that explains what is the purpose of this file? We can still get that info from the git log, but it's easier in a comment :)

Reply:
Also a README entry would be nice to allow people to discover this file/feature